### PR TITLE
added future DOB validation

### PR DESCRIPTION
**Pull request by Mudiwa66**
Users who enter a future date of birth will get the appropriate error message.

### CODE REVIEW

File: go-app-ussd_clinic_rapidpro.js [hunk 1]
```diff
@@ -911,13 +911,19 @@ go.app = function() {
...
+                        );
+                    }
                     if (!date.isBetween(current_date.clone().add(-2, "years"), current_date.add(1, "days")) ) {
                         return $(
                             "Unfortunately MomConnect doesn't send messages to children older " +
                             "than 2 years. Please try again by entering the day the baby was " +
                             "born as a number, e.g. 12."
                         );
-                    }
+                        }
```
Comment (erikh360) [inline go-app-ussd_clinic_rapidpro.js]: This new indentation looks weird

Comment (erikh360) [reply go-app-ussd_clinic_rapidpro.js]: Might just need to run npm test again

**Review by erikh360 — COMMENTED** [review]
Looks good just the one comment

File: go-app-ussd_clinic_rapidpro.js (hunk 1 above)
Comment (Mudiwa66) [reply go-app-ussd_clinic_rapidpro.js]: should be okay now

**Review by erikh360 — APPROVED** [review]

**Pull request merged**